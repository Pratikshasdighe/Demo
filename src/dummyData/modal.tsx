interface Iprops {
  heading: string;
  content: string;
}

const dummyData: Array<Iprops> = [
  {
    heading: "Calender month",
    content: "Ex:-March-31days,April-30days etc",
  },
  {
    heading: "Every month 30 days",
    content: "Ex:-March-31days,April-30days etc",
  },
];
export default dummyData;
