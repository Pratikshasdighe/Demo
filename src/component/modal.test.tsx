import React from "react";
import renderer from "react-test-renderer";
import { NativeBaseProvider } from "native-base";
import Model from "./modal";
import dummyData from "../dummyData/modal";

it("expect to render Modal", () => {
  const modalSnap = renderer
    .create(
      <NativeBaseProvider>
        <Model data={dummyData} />
      </NativeBaseProvider>
    )
    .toJSON();
  expect(modalSnap).toMatchSnapshot();
});
