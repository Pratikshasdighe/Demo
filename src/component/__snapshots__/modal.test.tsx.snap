// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Modal 1`] = `
<RNCSafeAreaProvider
  onInsetsChange={[Function]}
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
/>
`;
